index.html

```diff
@@ -0,0 +1,298 @@
+          <a href="#">My Page</a> 
+          <a href="#">Logout</a> 
+        </div>
+      </div>
+    </section>
+      
+    
+  <nav class="navbar navbar-expand-lg menu shadow fixed-top mt-5">
+    <div class="container">
+      <a class="navbar-brand" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav"
+        aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="" role="button"><i class="fa fa-bars" aria-hidden="true" style="color:#e6e6ff"></i></span>
+      </button>
+      <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link active" aria-current="page" href="#">Home</a></li>
+          <li class="nav-item"><a class="nav-link" href="pages/about.html">About</a></li>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <!--########################################
+               INTRO SECTION
+    #############################################-->
+
+  <section id="home" class="intro-section">
+    <div class="img">
+
+    </div>
+    <div class="container">
+      <div class="row align-items-center">
+        <div class="col-md-6-intros">
+          <h1 class="display-2">
+            <span class="display-2--intro">
+              "Hello! Sharing world"
+            </span>
+            <span class="display-2--description">
+              SECOND EDITION<br> 
+              OF THE FITNESS<br> 
+              CAMPAIGN
+            </span>
+          </h1>
+        </div>
+        <div class="col-md-6-intros">
+          <div class="mt-3 mb-3">
+            <div class="alert alert-secondary w-50 p-3 border border-warning shadow-sm" role="alert">
+              When you're at the gym feeling like you'll never be one of those people who look like they've 
+              been at it their entire lives, remember that they all started somewhere.
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6-intros">
+          <h1 class="display-2">
+            <span class="display-2--schedule">
+              (MON) ~ (FRI)
+            </span>
+            <span class="display-2--locale">
+              @ National Fitness Academy
+            </span>
+          </h1>
+        </div>
+
+      </div>
+    </div>
+  </section>
+
+  <!--########################################
+               MAIN PROGRAM SECTION
+    #############################################-->
+  <section class="main-program">
+    <div class="container d-flex justify-content-center">
+      <div class="row text-center d-flex mt-3">
+        <h4 class="fw-bold lead mb-3 text-white">MAIN PROGRAM</h4>
+        <div class="heading-line mb-5"></div>
+      </div>
+    </div>
+    <div class="container d-flex">
+      <div class="row d-flex justify-content-center">
```

Comment:
- [ ] Please In your main program section, add a title, and description, and a line of text at the bottom
- Your design:
![1](https://user-images.githubusercontent.com/63972670/176971904-0a151341-bce7-4d97-a848-8449d49a2eee.PNG)
- Original Design
-
![a](https://user-images.githubusercontent.com/63972670/176971949-2ef47711-709d-4bc6-a7ec-ad6bf8880948.png)

Reply:
Done